metadata.json

```
@@ -67,7 +67,7 @@
     },
     {
       "name": "puppet/python",
-      "version_requirement": ">= 1.12.0 < 4.0.0"
+      "version_requirement": ">= 1.12.0 < 5.0.0"
```

The latest is 5.0.0 so should it be
```suggestion
      "version_requirement": ">= 1.12.0 < 6.0.0"
```